bundles/org.openhab.binding.automower/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="automower"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Automower Bridge Thing Type -->
+	<bridge-type id="bridge">
+		<label>Automower Connect Bridge</label>
+		<description>The bridge to communicate with the Automower Connect API</description>
+
+		<config-description>
+			<parameter name="appKey" type="text" required="true">
+				<label>Application Key</label>
+				<description>The Application Key is required to communication with the Automower Connect Api. It can be obtained by
+					registering an Application on the Husqvarna Website. This application also needs to be connected to the
+					"Authentication API" and the "Automower Connect API"</description>
+			</parameter>
+			<parameter name="userName" type="text" required="true">
+				<label>User Name</label>
+				<description>The user name for which the application key has been issued</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<context>Password</context>
+				<description>The password for the given user</description>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" required="false" unit="s">
+				<label>Polling Interval</label>
+				<default>3600</default>
+				<description>How often the available automowers should be queried in seconds</description>
+			</parameter>
+		</config-description>
+
+	</bridge-type>
+
+	<!-- Automower Thing Type -->
+	<thing-type id="automower">
+		<label>Automower</label>
```

Comment:
You could reference the bridge type here, to eliminate the checkstyle warning.

Reply:
How would I do this?

Reply:
With
```
		<supported-bridge-type-refs>
			<bridge-type-ref id="yourBridgeId"/>
		</supported-bridge-type-refs>
```

Reply:
Thanks!